LICENSE.md

```diff
@@ -6,15 +6,19 @@ Version 2.0.3
 0. Definitions
 --------------
 
+**Source Code**: The preferred form of the work for making modifications to it.
+
+**Object code**: Any non-source form of a work. 
```

Comment:
By this he means binaries. I agree there should be a better definition of this, but this is not the best definition of it. This goes along with below, For example, what if the language is not a compiled language, that means that author could argue that distributing the source is distributing the object code, thus blocking git and all else...tricky to get that right